code/__HELPERS/hearted.dm

```diff
@@ -1,8 +1,10 @@
 /// Called when the shuttle starts launching back to centcom, polls a few random players who joined the round for commendations
 /datum/controller/subsystem/ticker/proc/poll_hearts()
-	if(!CONFIG_GET(number/commendations))
+	if(!CONFIG_GET(number/commendation_percent_poll))
 		return
-	var/number_to_ask = round(LAZYLEN(GLOB.joined_player_list) * CONFIG_GET(number/commendations)) + rand(0,1)
+
+	var/number_to_ask = round(LAZYLEN(GLOB.joined_player_list) * CONFIG_GET(number/commendation_percent_poll)) + rand(0,1)
```

Comment:
What's the rand here for?

Reply:
I just threw that in there with lowpop servers in mind, in case they don't have enough players who joined the round to guarantee at least one commendation, there'd still be a chance for someone to be asked